packages/api/src/app-router.ts

```diff
@@ -260,7 +262,8 @@ export default async function makeApp(params: CliArgs) {
   if (fallbackProxy) {
     app.use(proxy({ target: fallbackProxy, changeOrigin: true }));
   }
-  app.use(errorHandler());
```

Comment:
I think we still need the errorHandler, it is not a "not found handler"!
https://expressjs.com/en/guide/error-handling.html#writing-error-handlers

Reply:
Good catch, I was about to break a lot of error handling. I went ahead and added it to `prefixRouter` instead of `app` so its behavior kicks in for all `/api` endpoints but Next.js can still handle other errors however it wants.